lib/reader/src/parser.rs

```diff
@@ -633,9 +631,16 @@ impl<'a> Parser<'a> {
                         Some(w) => w,
                     };
                     let mut isa_builder = match isa::lookup(isa_name) {
-                        None => return err!(loc, "unknown ISA '{}'", isa_name),
-                        Some(b) => b,
+                        Err(isa::LookupError::Unknown) => {
+                            return err!(loc, "unknown ISA '{}'", isa_name)
+                        }
+                        Err(isa::LookupError::Unsupported) => {
+                            continue;
+                        }
```

Comment:
This looks right to me, but it would be good to add a test that verifies the `IsaSpec` returned in the three cases:

1. Test file has no ISA specs.
2. Test file has valid ISA specs, but they are not supported.
3. Test file contains valid and supported ISA specs.